bindings/nodejs/lib/types/wallet/wallet.ts

```diff
@@ -1,14 +1,155 @@
+    /** The total amount of the outputs */
+    total: u64;
+    /** The amount of the outputs that aren't used in a transaction */
+    available: u64;
+    /** Voting power */
+    votingPower: string;
+}
+
+/** The required storage deposit per output type */
+export interface RequiredStorageDeposit {
+    /** The required amount for Alias outputs. */
+    account: u64;
+    /** The required amount for Basic outputs. */
+    basic: u64;
+    /** The required amount for Foundry outputs. */
+    foundry: u64;
+    /** The required amount for NFT outputs. */
+    nft: u64;
+}
+
+/** The balance of a native token */
+export interface NativeTokenBalance {
+    /** The native token id. */
+    tokenId: HexEncodedString;
+    /** Some metadata of the native token. */
+    metadata?: string;
+    /** The total native token balance. */
+    total: u256;
+    /** The available amount of the total native token balance. */
+    available: u256;
+}
+
+/** Sync options for an account */
+export interface SyncOptions {
+    /**
+     * Usually syncing is skipped if it's called in between 200ms, because there can only be new changes every
+     * milestone and calling it twice "at the same time" will not return new data
+     * When this to true, we will sync anyways, even if it's called 0ms after the las sync finished. Default: false.
+     */
+    forceSyncing?: boolean;
+    /// Try to sync transactions from incoming outputs with their inputs. Some data may not be obtained if it has been
+    /// pruned.
+    syncIncomingTransactions?: boolean;
+    /** Checks pending transactions and reissues them if necessary. Default: true. */
+    syncPendingTransactions?: boolean;
+    /** Specifies what outputs should be synced for the ed25519 addresses from the account. */
+    account?: AccountSyncOptions;
+    /** Specifies what outputs should be synced for the address of an account output. */
+    // TODO Rename when we are done with Account changes https://github.com/iotaledger/iota-sdk/issues/647.
+    alias?: AliasSyncOptions;
+    /** Specifies what outputs should be synced for the address of an nft output. */
+    nft?: NftSyncOptions;
+    /** Specifies if only basic outputs with an AddressUnlockCondition alone should be synced, will overwrite `account`, `alias` and `nft` options. Default: false. */
+    syncOnlyMostBasicOutputs?: boolean;
+    /** Sync native token foundries, so their metadata can be returned in the balance. Default: false. */
+    syncNativeTokenFoundries?: boolean;
+}
+
+/** Specifies what outputs should be synced for the ed25519 addresses from the account. */
+export interface AccountSyncOptions {
+    /** Whether to sync Basic outputs. */
+    basicOutputs?: boolean;
+    /** Whether to sync Account outputs. */
+    accountOutputs?: boolean;
+    /** Whether to sync NFT outputs. */
+    nftOutputs?: boolean;
+}
+
+/** Specifies what outputs should be synced for the address of an account output. */
+export interface AliasSyncOptions {
```

Comment:
This has been renamed in the Rust PR, want to do it as well?

Reply:
Done